MMM-pages.js

```diff
@@ -173,23 +174,51 @@ Module.register('MMM-pages', {
 
     // Hides all modules not on the current page. This hides any module not
     // meant to be shown.
+
+var subclassBy = this.config.subclassBy;
+var majorModules = this.config.modules[this.curPage].map(function(a){ return a.split('.')[0]});
+var minorModules = this.config.modules[this.curPage].map(function(a){ var b = a.split('.')
+ if (b.length>1){return {major:b[0], minor: b[1]}} else {return {major:a}};
+});
+
     MM.getModules()
       .exceptWithClass(this.config.fixed)
-      .exceptWithClass(this.config.modules[this.curPage])
+      .exceptWithClass(majorModules)
       .enumerate(module => module.hide(
         self.config.animationTime / 2,
         { lockString: self.identifier }
       ));
 
+      var shortlist = MM.getModules()
+      .withClass(majorModules);
+    
+      shortlist.enumerate((module) => {
+        var mdata = module.data;
+        if (minorModules.find( function(el) { 
+          return (mdata.name === el.major) && (!el.minor || (mdata[subclassBy] === el.minor))} 
+          )) {
+          
+        console.log("should not hide this");
```

Comment:
Magic Mirror modules don't use `console.log`. In addition, this would be a very noisy log statement—I don't think we should have a log statement here at all.